web/src/utils/formatting.ts

```diff
@@ -11,18 +11,29 @@ function addSpaceBetweenNumberAndUnit(inputString: string) {
 
 export const formatEnergy = function (
   d: number,
-  numberDigits: number = DEFAULT_NUM_DIGITS
+  numberDigits: number = DEFAULT_NUM_DIGITS,
+  isCapacity = false
 ) {
   // Assume MW input
   if (d == undefined || Number.isNaN(d)) {
     return d;
   }
   const significantFigures = d.toString().length > 1 ? numberDigits : 1;
-
-  const power = `${d3.format(`.${significantFigures}s`)(d * 1e6)}Wh`;
+  const unit = isCapacity ? 'W' : 'Wh';
+  const power =
+    d < 1e9
+      ? d3.format(`.${significantFigures}s`)(d * 1e6) + unit
+      : d3.format(`.${significantFigures}r`)(d / 1e6) + 'TWh';
   return addSpaceBetweenNumberAndUnit(power);
 };
 
+export const formatPower = function (
+  d: number,
+  numberDigits: number = DEFAULT_NUM_DIGITS
+) {
+  return formatEnergy(d, numberDigits, true);
+};
+
```

Comment:
Just had a thought: how about flipping the two functions around, so `formatPower` is the default that does the d3 logic, and then `formatEnergy` just adds "h" to the output of the `formatPower`?

```js
export const formatPower = function (
  d: number,
  numberDigits: number = DEFAULT_NUM_DIGITS
) {
  return `${formatEnergy(d, numberDigits)h`;
};
```